src/digitalIdentity-backend/pom.xml

```diff
@@ -58,6 +58,10 @@
 			<artifactId>javase</artifactId>
 			<version>3.3.0</version>
 		</dependency>
+		<dependency>
```

Comment:
We don't need this dependency! It is already added... Because of this we have the weird warning:


```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for didentity.amos:digitalIdentity:jar:0.0.1-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.springframework.boot:spring-boot-starter-web:jar -> duplicate declaration of version (?) @ line 61, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```